### ISSUE TITLE
You plan to use virtualization in a MessageList component?

### ISSUE THREAD

**Issue by francesc79** [chatscope/chat-ui-kit-react#90]
You plan to use virtualization in a MessageList component?

**Comment by supersnager** [comment]
@francesc79 Yes, I thought about it, but there is no exactly planned when it will be done.
However, you can try to use https://github.com/bvaughn/react-window.

I know that @GhazanfarKhan used it with MessageList: https://github.com/chatscope/chat-ui-kit-react/issues/45#issuecomment-875504590
